kroma-validator/flags/flags.go

```diff
@@ -82,15 +82,15 @@ var (
 		EnvVars:  prefixEnvVars("CHALLENGER_ENABLED"),
 		Required: true,
 	}
-	ChallengerPollIntervalFlag = &cli.DurationFlag{
-		Name:     "challenger.poll-interval",
-		Usage:    "Poll interval for challenge process",
-		Required: true,
-		EnvVars:  prefixEnvVars("CHALLENGER_POLL_INTERVAL"),
-	}
 
 	// Optional flags
 
+	ChallengePollIntervalFlag = &cli.DurationFlag{
+		Name:    "challenge-poll-interval",
+		Usage:   "Poll interval for challenge process",
+		EnvVars: prefixEnvVars("CHALLENGE_POLL_INTERVAL"),
```

Comment:
Couldn’t it be confusing since it differs from the names of other challenger environment variables?

Reply:
This config is used in both l2_output_submitter and challenger, so I changed it not to be misunderstood that this flag is only for challenger.